### PR TITLE
Cover ignored blocks in Spec and FAQ

### LINKED ISSUE #93: Cover ignored blocks
We should cover fsfe/reuse-tool#463 in the spec and mention, that REUSE information wrapped in the ignore strings is actually also ignored by the spec.

Also, create an FAQ item as described in fsfe/reuse-tool#465.

### PR DESCRIPTION
**Pull request by mxmehl**
Fixes #93

I am not sure how much we want to go into detail in the spec. Probably a better wording could spare us the two example but I am not creative enough right now.

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -113,6 +113,45 @@ testing.
 If you have an entire directory that you want to "exclude" from REUSE compliance
 testing, you can [use a DEP5 file](#bulk-license).
 
+## How do I exclude certain files from REUSE compliance testing? {#exclude-lines}
+
+Some files may contain commands or documentation that confuse the REUSE helper tool:
```
Comment (carmenbianca) [inline faq.md:118]: Answer the question first, provide context afterwards.

**Review by carmenbianca — CHANGES_REQUESTED** [review]

File: faq.md (hunk 1 above)
Comment (mxmehl) [reply faq.md]: Good point. I've reordered the answer a bit.

File: faq.md [hunk 2]
```diff
@@ -113,6 +113,49 @@ testing.
...
 
+## How do I exclude certain files from REUSE compliance testing? {#exclude-lines}
+
+In order to make the tool ignore a specific section containing strings that may
+falsely be detected as copyright or license statements, you can wrap it within
+the two comments `REUSE-IgnoreStart` and `REUSE-IgnoreEnd`.
+
+Please note that this MUST NOT be used to ignore valid copyright and licensing
+information by yourself or a third party. The ignore blocks MUST only be used
+for marking blocks that may trigger false-positive detections and errors.
```
Comment (carmenbianca) [inline faq.md:124]: I would remove the RFC-2119 capitalised 'MUST' and 'MUST NOT' from the FAQ.

Comment (mxmehl) [reply faq.md]: Done

**Review by carmenbianca — APPROVED** [review]
LGTM

**Pull request merged**